jazz_environment-event-handler/index.js

```diff
@@ -0,0 +1,561 @@
+
+                                    var environmentPayload = {};
+
+                                    if (results.checkInterest.interested_event) {
+                                        var payload = results.checkInterest.payload.Item;
+                                        logger.info("payload.SERVICE_CONTEXT.S:" + payload.SERVICE_CONTEXT.S);
+                                        var serviceContxt = JSON.parse(payload.SERVICE_CONTEXT.S);
+
+                                        if (serviceContxt === undefined) {
+                                            logger.debug("Service Context is not defined");
+
+                                            innerCallback({
+                                                error: "Service Context is not defined."
+                                            });
+                                        }
+
+                                        var username = payload.USERNAME.S;
+                                        if (username) {
+                                            serviceContxt.created_by = username;
+                                        }
+
+                                        logger.info("Username:" + username);
+                                        var service_name = payload.SERVICE_NAME.S;
+                                        if (service_name) {
+                                            serviceContxt.service_name = service_name;
+                                        }
+
+                                        var required_fields;
+                                        var missing_required_fields;
+
+                                        if (results.checkInterest.event_name === event_config.DELETE_ENVIRONMENT) {
+                                            required_fields = configData.ENVIRONMENT_UPDATE_REQUIRED_FIELDS;
+                                            // validate required fields
+                                            missing_required_fields = _.difference(_.values(required_fields), _.keys(serviceContxt));
+                                            if (missing_required_fields.length > 0) {
+                                                // return inputError
+                                                innerCallback({
+                                                    error: "Following field(s) are required - " + missing_required_fields.join(", ")
+                                                });
+                                            } else {
+                                                if (serviceContxt.endpoint !== undefined || serviceContxt.endpoint !== null || serviceContxt.endpoint !== "") {
+                                                    environmentPayload.endpoint = serviceContxt.endpoint;
+                                                }
+												
+												var event_status = payload.EVENT_STATUS.S;												
```

Comment:
fix indentation line 179 - 187

Reply:
This seems an issue in the GIT web UI only